### PR TITLE
Use standalone prettier

### PR DESCRIPTION
**Pull request by ntkoopman**
This allows using ts-poet using one of the CDN repository mirrors. The reason is that Prettier uses the standalone version on those, and the standalone version doesn't include `resolveConfig`.

```js
import("https://esm.sh/ts-poet").then(module => module.code("")) 
```
```
Uncaught (in promise) TypeError: K.resolveConfig is not a function
```

(mainly for Deno compatibility)

### CODE REVIEW

File: src/Code.ts [hunk 1]
```diff
@@ -209,12 +210,12 @@ export function deepGenerate(object: unknown): string {
   return result;
 }
 
-const configPromise = resolveConfig('./');
+const configPromise = resolveConfig?.('./');
```
Comment (stephenh) [inline src/Code.ts:215]: Huh, interesting. Sgtm. Can you include an in-source comment here that mentions the CDN/Deno use case and maybe links to this PR?

**Review by stephenh — APPROVED** [review]
Cool, thanks!

If this makes ts-poet work in Deno, which seems cool, is that worth mentioning in the readme? Not sure if that's standard practice / useful or not; I've not really done anything with Deno yet.

**Comment by ntkoopman** [comment]
The standard practice is to add dependencies to deno.land (https://deno.land/x), but that would need an extra build step. 

If you want to document support it's probably easier to document usage from a browser, since that would imply that it works in Deno as well.

**Comment by stephenh** [comment]
Looks great, thanks @ntkoopman ! I published this to npm was version 4.9.0.

**Pull request merged**